addons/web/static/src/core/pager/pager.js

```diff
@@ -76,7 +87,8 @@ export class Pager extends Component {
      */
     parse(value) {
         let [minimum, maximum] = value.trim().split(/\s*[-\s,;]\s*/);
-        const clamp = (value) => Math.min(Math.max(value, 1), this.props.total);
+        this.updateCounter();
```

Comment:
as above, this should probably we marked with `await`, so the rest of the code takes the complete value

Reply:
Same comment than above.